experiments/what-factors-caused-WDTK-to-grow.md

```diff
@@ -0,0 +1,45 @@
+
+So, Gemma interviewed one of the WhatDoTheyKnow’s founders, Francis Irving, specifically about what they did to promote the site. 
+
+## Answer
+
+Summary of most important things:
+
+
+
+- Getting a **Google grant to do Google Adwords**. Advice: Make sure you appear on the search page of when people are searching for FOI especially. Before Google Adwords WDTK did not appear at all on search results, then after they got the grant they were listed above government adverts/sites. 
+
+
+- Tom Steinberg had lots of **contacts in the media** so got the site featured in mainstream media
+
+
+- mySociety had built up a **strong community of people interested in civic tech and FOI** so there was a **desire for a site like WDTK well before it was built**. This community was built up over a number of years, through newspapers, blogs, mailing lists. mySociety also did a Call for Proposals for civic tech projects when charity first launched - based on an article Tom wrote about the concept of digital democracy etc. WDTK was suggested by Francis and someone else in the 2nd call. The best project ideas were taken and an event for 50-100 people was held to launch ideas. So community was already invested in the idea before it was even built. 
+
+
+- In UK **FOI was a new thing** at the time they launched - had more media buzz and interest. 
+
+
+- The **internet in general was pretty new**, so the thought of using it to request information and publish responses was very novel. 
+
+
+- They **approached lots of other FOI enthusiasts** - Campaign for FOI etc to tell them about site. Who are the equivalents to Ben Goldacre in other countries? 
+
+
+- Via Tom’s media contacts they got **various articles in the press**. Examples include: Highlighting authorities who used ridiculous excuses not to reply and had to be chased loads of times (ridiculing them basically and quoting individual FOI officers); silly requests got a lot of coverage (although could be seen as devaluing FOI)
+
+
+- They didn’t send out newsletters or anything to users. **Users found like-minded people working on similar issues via the Comments on requests, and then directly messaged each other**.
+
+
+- They had a **really good team of people with a variety of skills/interests**. Tom was really good at the marketing/promotion of the site, and it took him a considerable amount of time. So making sure Alaveteli partners have someone dedicated to marketing is really important. Advice: Try and find a volunteer who would find marketing the site an interesting challenge. 
+
+
+- There wasn’t any social media back then really. So they **used mailing lists and blogs, and mainstream media**. Mainstream media was easier to exploit in those days - less of a corporate hold. 
+
+Here are the [full notes](https://docs.google.com/document/d/16sUvKLLi6RnFYCjh1UryXuZK-lic5O-4kRkqxdkBz6w/edit) from our chat.
```

Comment:
Probably shouldn't be an edit link, have changed the Google doc permissions to "Anyone with the link can view" (it's in a mySoc folder tho so mySociety staff with access to the folder can still edit)